gateway/ln-gateway/src/lib.rs

```diff
@@ -911,7 +909,14 @@ impl Gateway {
                 .build(gw_client_cfg.clone(), self.clone())
                 .await?;
 
-            let federation_info = self.make_federation_info(&client, federation_id).await;
+            // Instead of using `make_federation_info`, we manually create federation info
+            // here because short channel id is not yet persisted
```

Comment:
I'm not sure I understand this comment. Doesn't `make_federation_info` do the same thing?

Reply:
Oh I see `make_federation_info` has changed